main.py

```diff
@@ -131,12 +134,13 @@ def __render_main(matrix, data):
 
 if __name__ == "__main__":
     # Check for led configuration arguments
-    matrixOptions = led_matrix_options(args())
+    command_line_args = args()
+    matrixOptions = led_matrix_options(command_line_args)
 
     # Initialize the matrix
     matrix = RGBMatrix(options=matrixOptions)
     try:
-        main(matrix)
+        main(matrix, command_line_args.config)
```

Comment:
I'm trying this now and I'd love if it could accept either `--config config-nats` or `--config config-nats.json`, so it works with tab-completion.

Can be done with
```suggestion
        config, _ = os.path.splitext(command_line_args.config)
        main(matrix, config)
```